opencraft/urls.py

```diff
@@ -37,11 +37,11 @@
     url(r'^admin/', admin.site.urls),
     url(r'^api/', include('api.urls', namespace='api')),
     url(r'^instance/', include('instance.urls', namespace='instance')),
-    url(r'^registration/', include('registration.urls', namespace='registration')),
+    url(r'^login/$', LoginView.as_view(authentication_form=LoginForm), name='login'),
+    url(r'^logout/$', LogoutView.as_view(next_page='/'), name='logout'),
     url(r'^reports/', include('reports.urls', namespace='reports')),
-    url(r'^email-verification/', include('email_verification.urls', namespace='email-verification')),
     url(r'^favicon\.ico$', RedirectView.as_view(url='/static/img/favicon/favicon.ico', permanent=False)),
-    url(r'^$', views.IndexView.as_view(), name='index'),
+    url(r'^$', RedirectView.as_view(url='/login'), name='index'),
```

Comment:
The index view should be kept around and still needs to have some logic to where redirect the users to.
Unlogged user -> Login form
Logged user (admin) -> Console
Betatest user -> New registration form

Reply:
I am unsure about how to redirect them to new registrations form :confused: 
Currently, I am just checking the user and reversing them back too appropriate places.
```
user = self.request.user
if InstanceReference.can_manage(user):
    return reverse('instance:index')
else:
    return reverse('login')
```